multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt

```diff
@@ -455,6 +455,7 @@ class MadaraGenerator : ThemeSourceGenerator {
         SingleLang("مانجا ليك", "https://mangalek.com", "ar", className = "Mangalek", overrideVersionCode = 2),
         SingleLang("مانجا لينك", "https://mangalink.io", "ar", className = "MangaLinkio", overrideVersionCode = 3),
         SingleLang("Chunchunmaru Translations", "https://chunchunmarutl.com", "es", className = "ChunchunmaruTranslations"),
+        SingleLang("فالكون مانجا", "https://falconmanga.com/", "ar", className = "FalconMangaAr"),
```

Comment:
```suggestion
        SingleLang("فالكون مانجا", "https://falconmanga.com", "ar", className = "FalconMangaAr"),
```

No need for trailling "/"

Reply:
Good catch, need fix this as well.